### PR TITLE
Swagger API documentation

### PR DESCRIPTION
**Pull request by robfletcher**
- [ ] add `"io.swagger:swagger-annotations:1.5.20"` to Kork
- [ ] add `"org.reflections:reflections:+"` to Kork

Addresses #230

### CODE REVIEW

File: keel-api/src/main/kotlin/com/netflix/spinnaker/config/SwaggerConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+@Configuration
+@EnableSwagger2
+class SwaggerConfiguration {
+  @Bean
+  fun api(resolver: TypeResolver, handlers: List<ResourceHandler<*, *>>): Docket =
+    Docket(SWAGGER_2)
+      .apply {
+        handlers
+          .map { it.supportedKind.second }
```
Comment (emjburns) [inline keel-api/src/main/kotlin/com/netflix/spinnaker/config/SwaggerConfiguration.kt]: I think we should map by api version + supportedKind (so we don't end up with this issue again: https://github.com/spinnaker/keel/issues/527)

File: keel-api/src/main/resources/swagger-api.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+swagger.api:
```
Comment (luispollo) [inline keel-api/src/main/resources/swagger-api.yml]: So we couldn't use Swagger code annotations to specify the info that's in this file?

Comment (robfletcher) [reply keel-api/src/main/resources/swagger-api.yml]: I'd rather have it in this file.

**Comment by robfletcher** [comment]
Superseded by #811

**Pull request closed without merging**